src/services/weatherService.js

```diff
@@ -11,7 +11,7 @@ export const getLocationData = async location => {
     );
   } catch (error) {
     // to be handled accordingly elsewhere
-    return Promise.reject(error);
+    throw error;
```

Comment:
Your variant was fine though, I didn;t mean to enforce my suggestion :) BTW if you are not handling error in any way here  - then you could jsut omit the catch part and thus your error state will fall through the flow to the next catch itslef. We will talk about it more on our next session.